stable/gangway/values.yaml

```diff
@@ -30,6 +30,8 @@ gangway:
   # Env var: GANGWAY_HOST
   # host: 0.0.0.0
 
+  serviceAccountName: default
+
```

Comment:
I’d leave this as the empty string otherwise it’ll try and create a service account called default. Otherwise it’s all good!

Reply:
Actually thats what this line is for (to prevent a "default" account from being created): https://github.com/helm/charts/pull/23865/files#diff-12877716fe599f7f45547e2204b44bf6R1

Reply:
@rk295 , I have changed it to use an empty string.